recipe/meta.yaml

```diff
@@ -1,32 +1,35 @@
 {% set name = "jupyterlab-geojson" %}
-{% set version = "3.3.1" %}
+{% set version = "3.4.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyterlab-geojson-{{ version }}.tar.gz
-  sha256: 45da6d1df8fedf8edf3fc6bdf58106d99d7b4a982df978342512b37c75393825
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyterlab_geojson-{{ version }}.tar.gz
+  sha256: cc7f6015f75f7816240402880134cd7174af64fbef010b1556071729f98a291f
 
 build:
   number: 0
   # jupyterlab is missing on s390x
-  skip: true  # [py<36 or (linux and s390x)]
-  script: {{ PYTHON }} -m pip install . -vv --no-deps
+  skip: true  # [py<38 or (linux and s390x)]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
-    - jupyter-packaging >=0.7.9,<1.0
-    - jupyterlab >=3.0.0,<4.0.0
     - pip
     - python
-    - setuptools
-    - wheel
+    - hatchling >=1.5.0
+    - hatch-nodejs-version
+    - hatch-jupyter-builder >=0.5
   run:
     - python
+  run_constrained:
+    - jupyterlab >=3,<5
```

Comment:
Should it be `jupyterlab>=4.0.0,<5`? See https://github.com/jupyterlab/jupyter-renderers/blob/4d08ef19cd90d56c121587f62764ee2a135caa98/packages/geojson-extension/pyproject.toml#L2

Reply:
From my understanding no, but I will verify it being functional with jupyterlab3 before merging.

Reply:
confirmed working fine with both